src/System.Management.Automation/engine/hostifaces/HostUtilities.cs

```diff
@@ -22,7 +22,10 @@ internal enum SuggestionMatchType
     {
         Command = 0,
         Error = 1,
-        Dynamic = 2
+        Dynamic = 2,
+
```

Comment:
I think the newline here makes the enum less cohesive to read. My feeling is that the values should either all have newlines (and comments) or all be together.

Reply:
Will add comments to each value